### PR TITLE
Add shortlinks API section

### PR DESCRIPTION
**Pull request by jaxsonv-gb**
Adds the previously hidden API documentation surrounding Shortlinks and adds a section detailing the difference between the in-App delivery type and Shortlinks.

Additionally removes any trailing whitespace from the API blueprint and fixes a syntax error around the `contacts` field on `/campaign`.

### CODE REVIEW

**Review by adecastilho — APPROVED** [review]

File: apiary.apib [hunk 1]
```diff
@@ -652,11 +662,202 @@ A common question is whether or not to include multiple recipients on a single P
...
+- Designed to be embedded into an existing app with minimal Giftbit branding and with no additional message as part of the gift.
+- The redemption experience is more streamlined with only a single gift offer that is pre-selected.
+- Users are automatically directed to the selected card without the need for additional steps.
+- All gift offers are marked redeemed. They cannot be canceled once created nor will they expire.
+- Each API request generates a single gift.
+
+### Obtaining the Shortlink
+
+Once a Shortlink request has been made, and if there are sufficient funds, it will take a short period of time before the links become available.
+We recommend polling the links endpoint every 5 minutes until it is ready.
```
Comment (tyrelh) [inline apiary.apib:701]: Do we recommend every 5 mins? That seems like a long time

**Review by tyrelh — APPROVED** [review]
Just one comment but looks good

File: apiary.apib (hunk 1 above)
Comment (jaxsonv-gb) [reply apiary.apib:701]: @tyrelh This was ported over from the older documentation so not sure if we need that long.   In testbed it is almost instant but that might not be representative.  

Going to test this in another environment and see if we could maybe change this to 1-2 minutes or as short as 30 seconds.

Comment (jaxsonv-gb) [reply apiary.apib:701]: @tyrelh Just tested this, generated a gift link and it took about 20-30 seconds.  We could probably change this to 2-3 minutes but 5 minutes is consistent with the API message:

 
Screen Shot 2023-06-13 at 11 56 42 AM

Comment (tyrelh) [reply apiary.apib:701]: Leaving it as is is fine if it's consistent with what were telling folks elsewhere.

**Pull request merged**